src/main/js/util/fieldModelUtilities.js

```diff
@@ -191,9 +191,20 @@ export function createFieldModelErrorKey(fieldKey) {
     return fieldKey.concat('Error');
 }
 
+export function checkContextAndDescriptor(fieldModel, context, descriptorName) {
+    const copy = JSON.parse(JSON.stringify(fieldModel));
```

Comment:
Don't we have a utility for copying? Even if it does the same thing, it's probably best to reference that function incase we want to change the implementation universally